Packs/Absolute/Integrations/Absolute/Absolute.py

```diff
@@ -775,7 +780,7 @@ def parse_geo_location_outputs(response):
         parsed_device['Coordinates'] = geo_data.get('point', {}).get('coordinates')
         parsed_device['LocationTechnology'] = geo_data.get('locationTechnology')
         parsed_device['Accuracy'] = geo_data.get('accuracy')
-        parsed_device['LastUpdate'] = geo_data.get('lastUpdate')
+        parsed_device['LastUpdate'] = geo_data.get('lastUpdateDateTimeUtc')
```

Comment:
so lastUpdate is no longer returned from the API? also for us?

Reply:
no, they changed it to lastUpdateDateTimeUtc